app/components/DynamicText.js

```diff
@@ -0,0 +1,28 @@
+// Custome Text handling RTL and LTR text direction
+
+import React, { Component } from 'react';
+import { Text , StyleSheet, I18nManager} from 'react-native';
+
+
+class DynamicText extends Component {
```

Comment:
let's converge on Typography and make it RTL aware.

Reply:
Removed and now using Typography as the default app text